lib/api/objectPutPart.js

```diff
@@ -177,6 +180,31 @@ function objectPutPart(authInfo, request, streamingV4Params, log,
                         cipherBundle, splitter);
                 });
         },
+        // If data backend is real AWS, no need to store part metadata
```

Comment:
If data backend is backend that handles mpu (like real AWS), no need to store part info in metadata